components/layout_2020/flow/mod.rs

```diff
@@ -1766,7 +1766,7 @@ impl PlacementState {
                         .into(),
                     inline: Length::new(0.),
                 };
-                fragment.borrow_mut().adjust_offsets(offset);
+                fragment.borrow_mut().adjust_offsets(offset.into());
```

Comment:
The offset is created here by converting a `Length` into `Au`, so you could probably avoid the `.into()` here by simply creating a `LogicalVec<Au>` above.